src/containers/Confirmation.js

```diff
@@ -0,0 +1,48 @@
+//
+// Reggie Web
+//
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Shelly Xue Han on 2019-01-15.
+//
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Confirmation extends Component {
+  static displayName = '[Component Confirmation]';
+
+  render() {
+    return (
+      <div>
+        <h1>Welcome back,</h1>
+        <h2>{this.props.location.search}</h2>
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = state => {
```

Comment:
does this component need to be connected?

Reply:
will need it in next pr